src/soc/aspeed/ast25x0/src/asmram.rs

```diff
@@ -128,7 +127,7 @@ fn peek(a: u32) -> u32 {
     let y = a as *const u32;
     unsafe { ptr::read_volatile(y) }
 }
-pub fn ram(w: &mut print::WriteTo) -> () {
+pub fn ram(w: &mut impl core::fmt::Write) -> () {
```

Comment:
I wonder if it is this way for space reasons? I no longer recall. But I also think that we can make print::WriteTo fit it might be better?

Reply:
I have no context for the difference between these two, but there wasn't anywhere in soc where we used print::WriteTo that I could find;  I copied this core::fmt::Write call from soc/amd/common/boot/src/lib.rs;

https://github.com/oreboot/oreboot/blob/c66f625548f3439282edc95d89fa2a2f33799c21/src/soc/amd/common/boot/src/lib.rs#L80

Happy to change to whatever.